scripts/plugins/doc.py

```diff
@@ -8,9 +9,20 @@
 
 def Get_Doc(lookup_key):
     try:
-        msg = "\n\t{}\n\tFor more information checkout the Bitops Documentation: [{}]".format(
-            jh[lookup_key]["msg"], jh[lookup_key]["link"]
-        )
+        msg = "\n\t{}".format(jh[lookup_key]["msg"])
+        link = jh[lookup_key]["link"]
+        exit_code = 0
```

Comment:
I see you handle the `exit_code` default logic later with try/except and so this part becomes unused. Worth removing.

```suggestion
```